scripts/Install-VSCode.ps1

```diff
@@ -167,13 +175,21 @@ if (!($IsLinux -or $IsOSX)) {
             break;
         }
     }
+    if (($User -eq $true) -and ($BuildEdition -eq "Insider")) {
```

Comment:
You should be able to do just `if ($User -and ...)` here

Reply:
fixed and about to check in